compiler/lib/js_output.ml

```diff
@@ -127,10 +127,13 @@ struct
       | Some _ -> ()
 
   let ident f = function
-    | S {name; var = None} -> PP.string f name
-    | S {name; var = Some v} ->
-        output_debug_info_ident f name v;
+    | S {name; var = Some v; _} ->
```

Comment:
Here, we ignore `loc` and only try to extract a loc from `var`. I wonder if we should have a pass that try populate all the locs

Reply:
Can sourcemap accomodate multiple bindings for one identifier ?

I think a better longterm solution is to enforce (by datatype) that there is a loc in either `v` or `pi`.